h2o-assemblies/steam/Jenkinsfile

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/groovy
+
+@Library('test-shared-library') _
+
+def dockerImage
+def branchOrTag
+def steamImage
+
+pipeline {
+    agent { node { label 'linux&&docker' } }
+
+    parameters {
+        string(name: 'BRANCH_OR_TAG', defaultValue: 'master', description: 'Enter branch or tag you want to scan.')
+    }
+
+    options {
+        ansiColor('xterm')
+        timestamps()
+    }
+
+    stages {
+        stage('0. Init'){
+            steps{
+                script{
+                    checkout(
+                        scm: [$class: 'GitSCM',
+                            branches: [[name: '${BRANCH_OR_TAG}'], [name: 'devops/steam-prisma-jenkins']],
```

Comment:
do we need both branches here?

Reply:
Both branches were added to make sure that Dockerfiles exists irrespective of the input.